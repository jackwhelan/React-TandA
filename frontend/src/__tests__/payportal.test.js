import React from 'react';
import renderer from 'react-test-renderer';

import PayslipLog from '../components/payportal/PayslipLog.component';

describe('Pay Portal Components', () => {
    test('Payslip Log', () => {
        const component = renderer.create(<PayslipLog />);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    })
})