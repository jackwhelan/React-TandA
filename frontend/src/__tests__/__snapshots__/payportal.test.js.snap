// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pay Portal Components Payslip Log 1`] = `
<div
  className="core-cont bg-lightgrey mt-4"
>
  <h2>
    Loading...
  </h2>
  <hr
    className="bg-black"
  />
  <p>
    We are attempting to load news items from the database, if this problem persists, please contact a system administrator.
  </p>
</div>
`;
